lib/edools/enrollment.rb

```diff
@@ -46,5 +46,16 @@ def self.update(data = {})
     rescue Edools::RequestWithErrors => exception
       new exception.errors
     end
+
+    def self.every(data = {})
```

Comment:
This name is not clear enough , forcing us to read the documentation , neither the documentation is clear , what do you mean by "Entire query data" the entire set without pagination? unstructured data? structured data with every field?

What is exactly is the difference between the return of `all` and `every` , also every gives us the idea of a loop , is it an enumerable?

Reply:
I did not find a better name to represent the method.

So, the response in all method is handled for the attributes set of the Edools::Enrollment class.

where are they:
- id
- first_name
- email
- type
- errors

IMHO, I don't like this.

But not to break anything that depends on the all method, I created the every method.

if you have a better suggestion... I'm listening 🗣👂